docs/UserGuide.md

```diff
@@ -92,6 +92,141 @@ Some example commands you can try:
 `DIRECTION` refers to the direction the player wants the shot to follow as displayed on the graph.
 Format: `shoot 2`
 
+### Shoot Command: `shoot`
+
+#### Overview
+
+The `shoot` command is an integral part of the game's penalty shootout mechanics, allowing the player to attempt to shoot a penalty by choosing one of the available directions. This command pits the player's choice against the AI's shot direction, determining whether the penalty is scored or not.
+
+#### Usage
+
+When it's the AI's turn to save, you will be prompted to select a direction to shoot. Enter the shoot command followed by your chosen direction. For example:
+
+shoot[direction] - Choose a direction to take a shot. Directions are represented by numbers (e.g., 0 for left, 1 for center, and 2 for right --> for beginners).
+
+#### Understanding Player Skill Levels
+
+Your ability to score goals is significantly impacted by your player's skill level. As you progress through the game, you'll have the opportunity to upgrade your player from a Beginner to Medium and finally to an Expert. Each level enhances your player's effectiveness:
+
+- BeginnerSkillPlayer: Starts with basic shooting capabilities. Ideal for new players getting accustomed to the game mechanics.
+
+- MediumSkillPlayer: Offers improved shooting chances with a moderate increase in skill and power. Suitable for players looking for a balanced challenge.
+
+- ExpertSkillPlayer: Provides the best shooting abilities, significantly increasing your chances of getting your shots through the AI's saves. Designed for experienced players seeking the ultimate challenge.
+
+#### Difficulty Settings
```

Comment:
The illustration for the difficulty settings is great, I forgot to include that in the UG.